phpunit/class-rest-shortcodes-controller-test.php

```diff
@@ -137,46 +131,16 @@ public function data_get_item_with_field_combinations() {
 					'postId'    => self::$post_id,
 				),
 				array(
-					'html'  => 'Sorry, couldn\'t render a preview',
-					'type'  => 'html',
-					'style' => '',
-					'js'    => '',
-				),
-			),
-			// Youtube embed.
-			array(
-				array(
-					'shortcode' => '[embed]https://www.youtube.com/watch?v=8OBfr46Y0cQ[/embed]',
-					'postId'    => self::$post_id,
-				),
-				array(
-					'html'  => '<iframe width="600" height="338" src="https://www.youtube.com/embed/8OBfr46Y0cQ?feature=oembed" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>',
-					'type'  => 'video',
-					'style' => '',
-					'js'    => '',
-				),
-			),
-			// Vimeo embed.
```

Comment:
This also shouldn't be removed altogether. Maybe as part of another test suite, or maybe replacing the width & height attributes & simply wrapping iframe in a `<p>`.

I got these working, so check out https://github.com/Lewiscowles1986/gutenberg/tree/new-shoes